app/src/main/java/nl/tudelft/bbw/crawler/BlockDatabase.java

```diff
@@ -0,0 +1,73 @@
+package nl.tudelft.bbw.crawler;
+
+import android.content.Context;
+import android.database.sqlite.SQLiteDatabase;
+
+import com.readystatesoftware.sqliteasset.SQLiteAssetHelper;
+
+/**
+ * Database handler for the block database implementation of the crawler
+ * https://github.com/YourDaddyIsHere/even-cleaner-neighbor-discovery/blob/master
+ * /HalfBlockDatabase.py
+ */
+public class BlockDatabase extends SQLiteAssetHelper {
```

Comment:
This specific database extends SQLiteAssetHelper, which is used to read from an existing database. The other Database class which is used in our project, extends SQLiteOpenHelper, which is handy for creating databases. But I agree that the name is a bit confusing.